File: s10_Service/StickyServiceAudio/app/src/main/java/com/omeryildizce/stickyserviceaudio/MainActivity.java
package com.omeryildizce.stickyserviceaudio;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.media.MediaPlayer;
import android.os.Bundle;
import android.provider.Settings;
import android.view.View;

public class MainActivity extends AppCompatActivity {
    Intent intent;

    MediaPlayer mediaPlayer;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void play(View view) {
        intent = new Intent(this, StickService.class);
        startService(intent);
    }

    public void stop(View view) {
        intent = new Intent(this, StickService.class);

        stopService(intent);
    }
}

File: s04_AndroidTesting/TestIntro/app/src/main/java/com/omeryildizce/testintro/MainActivity.kt
package com.omeryildizce.testintro

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val tax = Tax()
        val income = tax.calculateIncome(100.0, 0.1)
        val netTax = tax.calculateTax(100.0,0.1)

        println(netTax)
        println(income)

    }
}